### ISSUE TITLE
Not Comparable Results

### ISSUE THREAD

**Issue by SII-Ferenas** [NVlabs/GroupViT#42]
Hi, I have trained GroupVIT under your default settings with RedCAP+GCC3M+GCC12M, and I ran the experiments 2 times, only achieving 26.8% mIoU and 27.9% mIoU. According to the previous discussions in the Closed Issue, I found that it may be caused by the training data size. So would you plz tell me the specific data size of RedCap, GCC3M, GCC12M, and YFC100M?

**Comment by xvjiarui** [comment]
Hi @Ferenas 

May I have your global batch size? It is crucial to have large enough batch size to make contrastive learning work. In GroupViT, we use 4096. 

The detailed size of each dataset could be found here 

https://github.com/xvjiarui/GroupViT/blob/dbaf9e85a6903c948df05f292d0b2091e9852f94/configs/default.yml#L11-L30

**Comment by SII-Ferenas** [comment]
Thanks for your feedback! I set the batch size as the default and use 8 GPUs to train the network, so the global batch size is 2048, which I think may not be the problem. But I found that the number of shards is different, even smaller, from yours. For instance, in GCC3M you have 436 shards but I only have 331 shards. Does it work fine? I actually want to know what the specific data shards storage in your device. For instance, GCC12M is reported to have 331GB but you say "GCC12M is 1.8T" [https://github.com/NVlabs/GroupViT/issues/16#issuecomment-1122995696].

**Comment by xvjiarui** [comment]
Hi @Ferenas 

It's due to saved with different image sized. But we are resizing to 224 during training, so it won't affect the result. 

But it's a common issue that GCC12M is not complete since many urls become invalid. You may need to check how many images you downloaded.

**Comment by SII-Ferenas** [comment]
Thanks for your reply, and I have downloaded the datasets again, found this is mainly due to the size of the datasets. BTW, I have one question about the convert_coco_object.py, could you please explain a little about the `clsID_to_trID`? Why do the label ids here need to be remapped?

**Comment by xvjiarui** [comment]
Hi @Ferenas 

It's to map the segmentation label into continuous integers.
